6/README.md

```diff
@@ -266,8 +266,23 @@ The endpoint must accept
 
 The response to a valid request must contain a JSON payload. 
 
-- It must return a map with the `status` key, the value of which must be one of [scheduled|running|succeeded|failed|unknown] 
-- Once the job has completed, and if it succeeded, it must also contain a map against the `result` key. The map with key(s) as defined in the `returns` section of the asset metadata.
+- It must return a map with the `status` key, the value of which must be one of [scheduled|running|succeeded|failed] 
+
+|Status |Description|
+|-|-|
+|scheduled| The job has been scheduled, but has not yet started|
```

Comment:
Add a cancelled status

Reply:
added, and updated the state transition diagram